src/v2/view-builder/views/ov/ResendView.js

```diff
@@ -1,8 +1,8 @@
-import { View, createCallout, _ } from 'okta';
+import { createCallout } from 'okta';
```

Comment:
We should change the name of this file now.. OVResendView

Reply:
done